doc/developer/diagnostic-questions.md

```diff
@@ -20,10 +20,14 @@ various ways that can cause Materialize to behave not as expected.
 
 # Checklist
 
+Is the user reporting the issue the same user that set up Materialize?
```

Comment:
I prefer a rewording of the question to "Are the user that is running Materialize, the user that set up Materialize, and the user reporting the issue all the same person?" since the three roles could be taken by different people. (A data analyst trying out Materialize, a DBA setting up Materialize, and a manager reporting the issue.)